From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 18:51:14 -0400
Subject: [net] myri10ge: compat header cleanups
Message-id: <20100510184858.2781.83846.sendpatchset@prarit.bos.redhat.com>
Patchwork-id: 24969
O-Subject: [RHEL5 PATCH 9/27] compat.h cleanup: myri10ge driver changes
Bugzilla: 546740
RH-Acked-by: Stanislaw Gruszka <sgruszka@redhat.com>

>From a438f6c061070f6668835579a387e589e13deb41 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Mon, 10 May 2010 09:32:04 -0400
Subject: [PATCH] myri10ge driver changes

Resolves 546740.

diff --git a/drivers/net/myri10ge/myri10ge.c b/drivers/net/myri10ge/myri10ge.c
index 8326926..c68fa0e 100644
--- a/drivers/net/myri10ge/myri10ge.c
+++ b/drivers/net/myri10ge/myri10ge.c
@@ -69,7 +69,6 @@
 #include <asm/mtrr.h>
 #endif
 
-#include "myri10ge_compat.h"
 #include "myri10ge_mcp.h"
 #include "myri10ge_mcp_gen_header.h"
 
diff --git a/drivers/net/myri10ge/myri10ge_compat.h b/drivers/net/myri10ge/myri10ge_compat.h
deleted file mode 100644
index 622e2de..0000000
--- a/drivers/net/myri10ge/myri10ge_compat.h
+++ /dev/null
@@ -1,9 +0,0 @@
-#ifndef __MYRI10GE_COMPAT_H__
-#define __MYRI10GE_COMPAT_H__
-
-/* RHEL5 compat */
-typedef __u16 __bitwise __sum16;
-typedef __u32 __bitwise __wsum;
-
-#endif /*  __MYRI10GE_COMPAT_H__ */
-
